### ISSUE TITLE
Improve Hover Info

### ISSUE THREAD

**Issue by Otifus** [mrbuilder1961/WheresMyChatHistory#19]
With the new feature of copying a message from a chat, by specifying an index, it would be extremely convenient if the index of the message was displayed when hovering over the Timestamp, this would make it easier to copy older messages that have rolled too high

**Comment by mrbuilder1961** [comment]
Unfortunately I won't be able to implement this without it adding an unnecessary amount of computational overhead and time wasted. This was hard to tackle because Minecraft uses 2 different Lists for chat messages: actual messages and the rendered ones. It's quite the pain to figure out which ones are related to each other, because there are always exceptions to when the values I used to check were present, and sometimes the (usually) constants would switch out from under me. I understand this is probably frustrating but I also have other things I need to work on and I don't want to waste more time on this issue.

TL;DR: I can't implement this without spending a lot more of my limited time, energy, brainpower, and hard drive space for such a minimal return. I apologize for the inconveniences, and if anyone has an idea on how to implement this comment on this thread of open a pull request.
